### DISCUSSION TITLE
Does it possible train a single pose model predict human body and animal at the same time?

### DISCUSSION THREAD

**Discussion by luohao123** [open-mmlab/mmpose#1110] [General] [answered]
Does it possible train a single pose model predict human body and animal at the same time?

Any instructions to follow if there no such model provided? Any dataset support both of them? I want detect pose on dog and human at the same time.

For detector I can using a coco trained model, and I will select dog and person class, but if there isn't such model I have to using 2 model which is very low efficient.

**Comment by ly015** [comment, marked as answer]
Maybe you can build a model with a shared backbone and multiple heads for human and animal keypoints respectively, and train it with a mixed dataset that gives both human and animal samples in each mini-batch. Relevant discussions  can be found at #662 and #757
